README.md

```diff
@@ -82,7 +82,7 @@ To compile AlphaRTC, please refer to the following steps
    git clone https://github.com/OpenNetLab/AlphaRTC.git
    ```
 
-3. Build Docker images
+3. Build Docker images(Please make sure you can get depottools)
```

Comment:
(Please make sure you can get depottools)  ->  Install depot_tools from the [official link](https://commondatastorage.googleapis.com/chrome-infra-docs/flat/depot_tools/docs/html/depot_tools_tutorial.html#_setting_up)

Reply:
depot_tools will be loaded automatically when compiling, but servers in some countries may not be able to access the site, resulting in compilation errors. This is to remind users to try to switch servers when they encounter this problem.

Reply:
Got it. Thank you for the elaboration!